### ISSUE TITLE
Help PO solve shared calendar issues

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#1646]
**Describe the issue**
The current shared cloud.pathfinder account has many people with (presumably) full access to it.  

Currently the PO schedules the team ceremonies from this calendar.  Several times now, the invites for the ceremonies have been updated with new recipients (new team members) and those changes have been reverted.  Also entire ceremonies have disappeared. 

This is frustrating for the team members that are caught by surprise by missing meetings or getting pulled in unexpectedly.  It's also leading to much rework and uncertainty for the PO.

**Additional context**
Add any other context, attachments or screenshots

**Definition of done**
- Sometime during the next couple weeks:
- Derive a list of all the people with access to cloud.pathfinder
- PO and Dana to review, decide appropriate permissions levels for each party (either remove if no longer needed or drop to view only if possible by default)
- Above access changes implemented.

**Comment by ActionAnalytics** [comment]
Outlook is now on my Windows machine, unblocked, starting to work on this

**Comment by ActionAnalytics** [comment]
This is not going anywhere, will take out of board
